taskcluster/scripts/pipeline/train-taskcluster.sh

```diff
@@ -38,11 +38,17 @@ case "$pretrained_model_mode" in
         ;;
     "continue"|"init"|"None")
         if [ "$pretrained_model_mode" == "None" ]; then
-            vocab="$MOZ_FETCHES_DIR/vocab.spm"
-        else
-            vocab="$TASK_WORKDIR/artifacts/vocab.spm"
```

Comment:
Now that we're copying, the vocab location is the same for all three of these modes.